test/tla/cli-integration-tests.md

```diff
@@ -3255,6 +3255,30 @@ Typing input error: Missing @typeAlias: newAlias = <type>;
 EXITCODE: ERROR (255)
 ```
 
+### typecheck TestUnsafeRecord.tla
+
+```sh
+$ $ apalache-mc typecheck TestUnsafeRecord.tla | sed 's/[IEW]@.*//'
+...
+[TestUnsafeRecord.tla:6:17-6:19]: Cannot apply R() to the argument "b" in R()["b"]
+```
+
+### typecheck TestGetX.tla
+
+```sh
+$ $ apalache-mc typecheck TestGetX.tla | sed 's/[IEW]@.*//'
+...
+[TestGetX.tla:2:12-2:14]: Cannot apply r to the argument "x" in r["x"].
+```
+
+### typecheck TestGetXWithRows.tla
+
+```sh
+$ $ apalache-mc typecheck TestGetXWithRows.tla | sed 's/[IEW]@.*//'
+...
+EXITCODE: OK
+```
```

Comment:
```suggestion
```sh
$ apalache-mc typecheck TestUnsafeRecord.tla | sed 's/[IEW]@.*//'
...
[TestUnsafeRecord.tla:6:17-6:19]: Cannot apply R() to the argument "b" in R()["b"]
```

### typecheck TestGetX.tla

```sh
$ apalache-mc typecheck TestGetX.tla | sed 's/[IEW]@.*//'
...
[TestGetX.tla:2:12-2:14]: Cannot apply r to the argument "x" in r["x"].
```

### typecheck TestGetXWithRows.tla

```sh
$ apalache-mc typecheck TestGetXWithRows.tla | sed 's/[IEW]@.*//'
...
EXITCODE: OK
```
```